Sources/Jetworking/Client/Client.swift

```diff
@@ -127,6 +128,29 @@ public final class Client {
         return nil
     }
 
+    @available(iOS 15.0, macOS 12.0, *)
```

Comment:
Consider moving the Version restricted parts into an extension.

Reply:
Restructured the `Client`.